### PR TITLE
fix(cli): [nan-1457] auto confirm for dry run

### PR DESCRIPTION
**Pull request by khaliqgant**
## Describe your changes
Honor the `--auto-confirm` option for dry run to automatically cycle through logs

## Issue ticket number and link
NAN-1457

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: .github/workflows/sync-integration-templates.yaml [hunk 1]
```diff
@@ -39,3 +39,4 @@ jobs:
               env:
                   AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
                   AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+                  AWS_DEFAULT_REGION: ${{ secrets.AWS_REGION }}
```
Comment (TBonnin) [inline .github/workflows/sync-integration-templates.yaml:42]: is that part of this PR?

File: packages/cli/lib/services/dryrun.service.ts [hunk 2]
```diff
@@ -368,9 +368,9 @@ export class DryRunService implements ScriptExecutorInterface {
 
                 while (index < logMessages.messages.length) {
                     const remaining = logMessages.messages.length - index;
-                    const confirmation = await promptly.confirm(
-                        `There are ${remaining} logs messages remaining. Would you like to see the next 10 log messages? (y/n)`
-                    );
+                    const confirmation = options.autoConfirm
+                        ? true
+                        : await promptly.confirm(`There are ${remaining} logs messages remaining. Would you like to see the next 10 log messages? (y/n)`);
```
Comment (TBonnin) [inline packages/cli/lib/services/dryrun.service.ts:373]: if there is a lot of records can it overload their terminal?

**Review by TBonnin — APPROVED** [review]

File: .github/workflows/sync-integration-templates.yaml (hunk 1 above)
Comment (khaliqgant) [reply .github/workflows/sync-integration-templates.yaml:42]: It probably should be a part of another PR but tried to squeeze it in 😅

File: packages/cli/lib/services/dryrun.service.ts (hunk 2 above)
Comment (khaliqgant) [reply packages/cli/lib/services/dryrun.service.ts:373]: Maybe? Feel like that is an edge case that we can live with for now.

**Pull request merged**